drift/app.py

```diff
@@ -21,12 +21,8 @@ def create_app():
     """
     app_name = os.getenv("APP_NAME", "drift")
     openapi_args = {"path_prefix": config.path_prefix, "app_name": app_name}
-    connexion_app = connexion.App(
-        __name__, specification_dir="openapi/", arguments=openapi_args
```

Comment:
I'd advise to keep the line limit to the previous value that was used to limit number of changes and use what you already are used to. (if these kind of changes were due to the line limit increase)

Reply:
The line limit is set to 100, that's the reason of this change.

Of course we can move and set to 120 col, several project use that (NumPy, SciPy, Pandas) 
Just for note:
- PEP says 79  (https://www.python.org/dev/peps/pep-0008/#maximum-line-length())
- Black set to default 88  (https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html?highlight=line-length#id2)
- Django set to 119 for the code and 79 for the doc string (https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)

Reply:
I tend to prefer the longer the better :)
I don't like having to jump through hoops to try to force things onto shorter lines.  On the other hand, we don't want to be too far from standard.  I'm used to using black now to figure it out I guess. 79 is too short, in my opinion.  I'd be ok with keeping to the previous value of 100, but I'm open to expanding it if others prefer longer.

Reply:
+1 to the old value 100

Reply:
ack, let stay with 100 (this PR change to it)